packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,345 @@
+import * as React from 'react';
+import { safeCreate, safeMount } from '@fluentui/test-utilities';
+import { resetIds, KeyCodes } from '@fluentui/utilities';
```

Comment:
Maybe we do this in a separate PR but we should also try to remove these dependencies as they are coming from v8 and not converged packages, but given that the tests are extensive lets keep that in a separate PR.